### PR TITLE
Updated all of learn except for tutorials.

### PR DESCRIPTION
**Pull request by blackbaud-brandonhare**
Updated everything EXCEPT tutorials. I am awesome at naming things.

### CODE REVIEW

File: src/app/learn/reference/cli-commands/index.html [hunk 1]
```diff
@@ -1,26 +1,54 @@
 <sky-demo-nav></sky-demo-nav>
-<div class="sky-container">
-    <h1>SKY UX CLI commands</h1>
-    <p>This reference describes the commands and options that are available in the SKY UX CLI.</p>
-    
+<stache
+  pageTitle="SKYUX CLI Commands"
```
Comment (Blackbaud-SteveBrush) [inline src/app/learn/reference/cli-commands/index.html]: "SKYUX" or "SKY UX"? It's "SKY UX" on their own docs...

File: src/app/learn/reference/cli-commands/index.html [hunk 2]
```diff
@@ -1,26 +1,54 @@
...
-        <li><p><code>skyux watch</code> — Runs the <code>test</code> command and watches the file system for changes. The <code>watch</code> command monitors changes in much the same way as the <code>serve</code> command, but for testing, the <code>watch</code> command is always running, much like debug mode in Visual Studio. If you change a test, the window is watching that and runs the test. This command is useful when writing a bunch of tests.</p></li>
-        <li><p><code>skyux version</code> or <code>skyux -v</code> — Returns the version of SKY UX CLI.</p></li>
-        <li><p><code>skyux</code>, <code>skyux help</code>, or <code>skyux -h</code> — Displays help information for SKY UX CLI arguments. This lists the commands and options that are available in the SKY UX CLI and provides descriptions.</p></li>
+      <li>
+        <p><code>skyux new</code> — Initializes a new local SKY UX project by pulling down the latest version of the template and installing the necessary dependencies. For information about the default items that the template provides in a SKY UX single-page application, see <a routerLink="/learn/reference/template-items">the template items reference</a>.</p>
```
Comment (Blackbaud-SteveBrush) [inline src/app/learn/reference/cli-commands/index.html]: Make this a `<stache-code>`...

File: src/app/learn/reference/cli-commands/index.html (hunk 1 above)
Comment (blackbaud-brandonhare) [inline src/app/learn/reference/cli-commands/index.html]: I wish I knew. I just copied it.

File: src/app/learn/reference/cli-commands/index.html (hunk 2 above)
Comment (blackbaud-brandonhare) [inline src/app/learn/reference/cli-commands/index.html]: BOOOOOO. Ok.

File: src/app/learn/reference/cli-commands/index.html (hunk 1 above)
Comment (Blackbaud-SteveBrush) [inline src/app/learn/reference/cli-commands/index.html]: But it's "SKY UX" originally (here, on this file)

File: src/app/learn/reference/cli-commands/index.html (hunk 2 above)
Comment (Blackbaud-SteveBrush) [inline src/app/learn/reference/cli-commands/index.html]: lol

File: src/app/learn/reference/cli-commands/index.html (hunk 1 above)
Comment (blackbaud-brandonhare) [inline src/app/learn/reference/cli-commands/index.html]: NUH UH!

Comment (blackbaud-brandonhare) [inline src/app/learn/reference/cli-commands/index.html]: I will update it.

**Pull request merged**